content/docs/en/getting-started/4-templates.md

```diff
@@ -20,17 +19,22 @@ It is also assumed that you have [set up your system for NativeScript developmen
 
 ## nativescript-vue/vue-cli-template
 
-Project link: [https://github.com/nativescript-vue/vue-cli-template](https://github.com/nativescript-vue/vue-cli-template)
+**Project link:** [https://github.com/nativescript-vue/vue-cli-template](https://github.com/nativescript-vue/vue-cli-template)
+
+### Features
```

Comment:
I think **Features** and **Usage** don't need to be headings, it adds extra links in the final page that look a bit messy

Reply:
I switched from headings to bold formatting with the latest commit :)